platforms/hyperledger-fabric/charts/osn_create_channel/values.yaml

```diff
@@ -0,0 +1,65 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+metadata:
+  #Provide the namespace for organization's peer
+  #Eg. namespace: org1-example-com
+  namespace: default
+  #Provide Fabric version
+  #Eg. version: 2.5.4
+  network:
+    version: 
+  images:
+    #Provide the valid image name and version for fabric tools
+    #Eg. fabrictools: hyperledger/fabric-tools:1.4.0
+    fabrictools: 
+    #Provide the valid image name and version to read certificates from vault server
+    #Eg. alpineutils: hyperledgerlabs/alpine-utils:1.0
+    alpineutils: 
+  #Provide the custom labels
+  #NOTE: Provide labels other than name, release name , release service, chart version , chart name , app.
+  #Eg. labels:
+  #      role: create_channel
+  labels:
+
+deployment:
+  annotations:
+
+vault:
+  #Provide the vaultrole for an organization
+  #Eg. vaultrole: org1-vault-role
+  role:
+  #Provide the vault server address
+  #Eg. vaultaddress: http://Vault-884963190.eu-west-1.elb.amazonaws.com
+  address:
+  #Provide the kubernetes auth backed configured in vault for an organization
+  #Eg. authpath: fra-demo-hlkube-cluster-org1
+  authpath:
+  #Provide the value for vault secretprefix
+  #Eg. adminsecretprefix: secret/...
+  adminsecretprefix:
+  #Provide the value for vault secretprefix
+  #Eg. orderersecretprefix: secret/...
```

Comment:
Same as above, a realistic key,value pair would be more informative.

Reply:
The adminsecretprefix value has been removed because it isn't being used. 
 The orderersecretprefix value  has been updated